test/gradualizer_db_tests.erl

```diff
@@ -0,0 +1,7 @@
+-module(gradualizer_db_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+import_module_test() ->
+    ?assertMatch(not_found, gradualizer_db:import_module(hello)),
+    ?assertMatch(ok, gradualizer_db:import_module(lists)).
```

Comment:
Looks good. I like the test. To test beam files with and without the AST, I guess we would have to add compiling to the test, but that's a bit more work. Maybe in the future then.